_demos/boxes.go

```diff
@@ -63,11 +63,11 @@ func main() {
 	tcell.SetEncodingFallback(tcell.EncodingFallbackASCII)
 	s, e := tcell.NewScreen()
 	if e != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", e)
+		fmt.Fprintf(os.Stderr, "Error getting screen %v\n", e)
```

Comment:
Changes in this file are not strictly necessary, and I'd like to leave them out.